srcpkgs/libcgroup/patches/musl-sterror_r.patch

```diff
@@ -17,6 +16,6 @@ GNU libc's if _GNU_SOURCE is defined.
 +		return "strerror_r() failed";
 +	}
 +#endif
- 
+
```

Comment:
please revert the changes on patches

Reply:
the file has been reverted as in the original

Reply:
Something is wrong, GitHub still shows the change. Are you sure you reverted the changes?